Eth-Crowdfund/eth_crowdfund_be
# Seeds contributors

## What's this PR do?
* Adds seeds for Contributors and CampaignContributors
* Updates how seedfile is ran 
* Seed database with `$ python3 manage.py seed`
## Where should the reviewer start? 
* src/seeds/seed.py
## How should this be manually tested?
* run `$ python3 manage.py seed` 
* Check database to see if it was seeded
